index.html

```diff
@@ -26,24 +24,259 @@ <h1>Hey there. I'm Richard</h1>
          </div>
           <div class="support">
             <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-  don’t hestiate to contact me.</p>
+          don’t hestiate to contact me.</p>
           </div>
           <div class="social-media">
             <a href="#"><i class="fab fa-twitter social"></i></a>
             <a href="#"><i class="fab fa-linkedin social"></i></a>
             <a href="#"><i class="fab fa-medium-m social"></i></a>
-            <!-- <a href="#"><i class="fab fa-medium social"></i></a> -->
             <a href="#"><i class="fab fa-github social"></i></a>
             <a href="#"><i class="fab fa-angellist social"></i></a>
-
           </div>
-
           <div class="arrow">
             <button  class="arrow-button" ><i class="fas fa-chevron-down fa-2x"></i></button>
           </div>
         </section>
+   </div>
+   <div class="grid-container">
+       <h3>My Recent Works</h3>
+       <hr>
+
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
```

Comment:
- [ ] The list of languages has a `ul` tag to group them, and `li` per icon. Please change it. Apply it to other projects.
- [ ] You must stick to the design as much as possible (e.g., font, colors, images, tests, margins).

- Languages have not  underline and centered, the button has not bordered on the original design. Please fix it. Apply it to other project sections too.

YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/148470711-59a54b72-fc42-4814-a049-4b7393926ed8.png)
___
ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/148470735-5524bf87-01f9-41af-897e-db04826ece61.png)